examples/internationalization/gen_l10n_example/lib/main.dart

```diff
@@ -23,8 +23,8 @@ class MyApp extends StatelessWidget {
         GlobalCupertinoLocalizations.delegate,
       ],
       supportedLocales: [
-        Locale('en', ''), // English, no country code
-        Locale('es', ''), // Spanish, no country code
+        Locale('en'), // English, no country code
+        Locale('es'), // Spanish, no country code
```

Comment:
Delete the country code phrase, since that variable isn't being used.

```suggestion
        Locale('en'), // English
        Locale('es'), // Spanish
```